src/hf_hydrodata/model/dataset.csv

```diff
@@ -18,4 +18,5 @@ scan,point_observations,Soil Climate Analysis Network,usda,,,,,point,UTC-00:00,,
 ameriflux,point_observations,Ameriflux,ameriflux,,,,,point,UTC-00:00,,
 noaa,forcing,NOAA Climate Data,noaa,,https://www.climate.gov/maps-data/dataset/daily-temperature-and-precipitation-reports-data-tables,6/1/22,,gridded,UTC-00:00,,
 jasechko_2024,point_observations,"Jasechko et al. 2024 water table depth dataset, subset to the CONUS2 domain",jasechko_2024,10.1038/s41586-023-06879-8,https://zenodo.org/records/10003697,1801,2022,point,,,
-modis,remote_sensing,Cloud-free MODIS on/off Snow Cover 2000-2017,nasa,10.1038/sdata.2018.300,10.1038/sdata.2018.300,3/10/2000,2/28/2017,gridded,UTC-06:00,,
\ No newline at end of file
+modis,remote_sensing,Cloud-free MODIS on/off Snow Cover 2000-2017,nasa,10.1038/sdata.2018.300,10.1038/sdata.2018.300,3/10/2000,2/28/2017,gridded,UTC-06:00,,
+fan_2013,point_observations,"Fan et al. 2013 water table depth dataset, subset to CONUS2 domain",fan_2013,10.1126/science.1229881,,01/01/1927,12/31/2009,point,,,

```

Comment:
Same comment about date formats here. Also why some the titles are in quotes, just curious about this?

Reply:
I will update the date here also. Good flag for the quotes - I think that was just a copy-paste typo.

Reply:
Actually I just talked to Bill and the quotes here are for when there are special characters in the dataset description. Otherwise, the .csv would view something like a comma as a delimiter.

Reply:
Oh, that makes sense. Thanks for the explanation.